src/util/apiUtils.js

```diff
@@ -185,32 +187,44 @@ export const parseApiResponse = requestUrl => ([response, body]) => {
  */
 export const buildRequestArgs = externalRequestOpts =>
 	({ endpoint, params, flags }) => {
-
-		// cheap, brute-force object clone, acceptable for serializable object
-		const externalRequestOptsQuery = JSON.parse(JSON.stringify(externalRequestOpts));
-
-		externalRequestOptsQuery.url = encodeURI(`/${endpoint}`);
-		externalRequestOptsQuery.jar = createCookieJar(externalRequestOptsQuery.url);
+		const dataParams = querystring.stringify(params);
+		const headers = { ...externalRequestOpts.headers };
+		let url = encodeURI(`/${endpoint}`);
+		let body;
+		const jar = createCookieJar(url);
 
 		if (flags) {
-			externalRequestOptsQuery.headers['X-Meetup-Request-Flags'] = flags.join(',');
+			headers['X-Meetup-Request-Flags'] = flags.join(',');
```

Comment:
What happens if we always set this header? Will it break? Maybe something like:

```
headers['X-Meetup-Request-Flags'] = (flags || []).join(',')
```

In general I'm trying to see if we can move in the direction of not branching logic for which headers get set, but see if it's possible to send a value that is ignored, or see if the API just ignores certain headers depending on the method.